wpilibc/shared/include/Commands/Command.h

```diff
@@ -66,8 +66,7 @@ class Command : public ErrorBase, public NamedSendable, public ITableListener {
   bool IsInterruptible() const;
   void SetInterruptible(bool interruptible);
   bool DoesRequire(Subsystem* subsystem) const;
-  typedef std::set<Subsystem*> SubsystemSet;
-  SubsystemSet GetRequirements() const;
```

Comment:
The whole point of the typedef is so user code can write "Command::SubsystemSet" instead of using the exact type.  Granted "auto" makes this less important.  As long as basic operations are named the same as std::set, user code is unlikely to break (even though it's technically an API change, this is almost-duck-typing).